RunFromSourceGuideForMac.md

```diff
@@ -0,0 +1,19 @@
+# Prerequisite Software
```

Comment:
Enemizer should work on mac os as well: https://github.com/Ijwu/Enemizer/releases

Reply:
I did not know that! It is now added to that list.

Reply:
@black-sliver Can you confirm that Enemizer is needed only under the conditions that:
a) the Mac user or someone they're playing with wants to do enemy randomization *in ALTTP* and not some other game, AND
b) the Mac user must also want to generate an enemized ALTTP seed locally rather than using the public website, before any Enemizer steps are needed?

If so, I think it's good to specify how this is an edge-case directly in the guide, so that people don't have to mess with it if it doesn't apply to them. (E.g. the enemizer github doesn't even say it's for ALTTP; although of course that's somewhat a separate issue, it does affect the guide.)

Reply:
It is only required for generating enemized lttp seeds yes, which is an extreme edge case. you don't need it to play enemized lttp

Reply:
I'll make this adjustment then. Thanks!